modules/tinymce/src/themes/silver/test/ts/webdriver/throbber/ThrobberTabbingTest.ts

```diff
@@ -56,17 +58,22 @@ describe('webdriver.tinymce.themes.silver.throbber.ThrobberTabbingTest', () => {
     await pAssertFocus();
   };
 
+  const pSetFocus = async (selector: string) => {
```

Comment:
Again, this is confusing/misleading as it's doing more than just focusing Can we please come up with another name?